shared/charts/gpii-flushtokens/templates/cronjob.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flushtokens.name" . }}
+spec:
+  schedule: "*/15 * * * *"
+  concurrencyPolicy: Allow
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "flushtokens.name" . }}
+        spec:
+          shareProcessNamespace: true
+          containers:
+          - name: gpii-flushtokens
+            image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+            command: [ "/bin/sh", "-c", "-x", "/app/scripts/dockerDeleteExpiredAccessTokens.sh" ]
+            env:
+            - name: GPII_COUCHDB_URL
+              value: '{{ .Values.couchdbUrl }}'
+            # Use '--deleteAll' to delete all of the access tokens
+            - name: DELETE_ALL
+              value: 'false'
+          - name: istio-proxy-manager
+            image: gcr.io/gke-release/istio/proxyv2:1.0.6-gke.3@sha256:1f6dac3dbf69c0b0d519398003b6572e0218f695f1be295a2ffff95df2395ed0
```

Comment:
If we can use universal here as well, that would be great!

I thought we had to use the `istio-proxy` image because the istio-proxy _user_ (user id `1337`) is set-up inside that image -- there's an `istio-proxy` in `/etc/passwd`, in `/etc/group`, and so on.  If one can use *any* image and still run as `istio-proxy|1337`, that's ... cool.  I didn't know that could be done.  I thought that the use of `securityContext.runAsUser` only applied to users already specified within the docker image.

Live and learn ... :-D

Reply:
:) haha. I think we both thought we needed to use `istio-proxy` for different reasons, but neither of those is really needed.

You can try this for example with local docker:
```
$ docker run -it --rm --user 1234  alpine                                                                                               

$$ whoami
whoami: unknown uid 1234

$$ ps -a
PID   USER     TIME  COMMAND
    1 1234      0:00 /bin/sh
    8 1234      0:00 ps -a
```
So you can see it's running as non-existing user (some stuff like `whoami` won't work obviously)

I agree that actually using the Universal image sounds like the easiest way to go.